models/users.js

```diff
@@ -215,6 +215,35 @@ const fetchUserImage = async (users) => {
   return images;
 };
 
+/**
+ * Add default status
+ * @returns {Promise<string[]>}
+ */
+const addDefaultStatus = async () => {
+  try {
+    const usersSnapshot = await userModel.get();
+    const users = [];
+    const addedDefaultStatusUsers = [];
+
+    usersSnapshot.forEach((doc) => users.push({ id: doc.id, ...doc.data() }));
+
+    for (const user of users) {
+      if (user.status === undefined) {
```

Comment:
```suggestion
      if (!user.status) {
```

Reply:
But we are checking if the field exists or not. with your suggested case it will take all falsy values. And when the property does not exist in an object it will return `undefined`.

Reply:
If we are sure status will always be undefined then your approach is correct
Btw is there any problem in checking for all falsy values?